includes/api/class-wc-rest-system-status-controller.php

```diff
@@ -86,6 +126,104 @@ public function get_items( $request ) {
 		return rest_ensure_response( $response );
 	}
 
+	/**
+	 * A list of modes that can be toggled via WC's system status screens
+	 *
+	 * @return array
+	 */
+	public function _get_modes() {
```

Comment:
Totally forgot about modes. I wonder if we need these or if we can intelligently offer them elsewhere or via constants. Not sure if they should be enabled externally. Maybe we can discuss in team chat today?